static/docs/commands-reference/move.md

```diff
@@ -59,16 +59,16 @@ name:
 
 ```dvc
 $ dvc move data.csv other.csv
-$ cat data.csv.dvc
+$ cat other.csv.dvc
 ```
 
-And here is the updated content of the `data.csv.dvc`:
+And here is the updated content of the `other.csv.dvc`:
 
 ```yaml
-md5: 3d1a3e5a5b662490e198d6a6ae84984b
+md5: df33f383592e7a399e106ba1cb487a8c
+wdir: .
```

Comment:
remove wdir, keep cache

Reply:
it's an outdated version.  already updated the latest one,